yarn.lock

```diff
@@ -35176,7 +35115,7 @@ swiper@4.5.1:
   languageName: node
   linkType: hard
 
-"terser@npm:^5.3.4, terser@npm:^5.5.1, terser@npm:^5.7.2":
```

Comment:
I assume those `yarn.lock` changes stem from the packages being moved over to `devDependencies` and those (removed) dependencies not being needed at runtime anymore?

Reply:
There were one or two dependencies which I removed completely because I couldn't find any reference to them in the code base. Without them, the builds still pass, so I think that's a good indication they weren't being used any more. So that's why some have moved out of here.

To my knowledge, all dev and normal dependencies will be installed when you run `yarn install`. I think the only time the runtime difference applies is when you're installing a package via npm or something like that. So in reality, every time anyone interacts with this package.json, it's in "dev" mode, be that locally or in CI. So all the dev dependencies will still be installed. Therefore, just moving them doesn't really make a huge difference immediately. The main goal is to better categorize dependencies to make dependency updates easier. So it really only has an impact on renovate -- at least, that's my assumption.